app/helpers/checks_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ChecksHelper
+end
```

Comment:
If you don't occupy it, you can delete it.

Reply:
It's actually necessary, thx!